packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -67,6 +67,11 @@ export class DataStores implements IDisposable {
 
     private readonly disposeOnce = new Lazy<void>(() => this.contexts.dispose());
 
+    // number of dataStores during loadContainer
+    public readonly containerLoadDataStoreCount: number;
```

Comment:
nit: When considering the shape of API this class exposes, if these are only expected to be used for logging, it might be clearer to do this:

```typescript
public readonly stats: {
  readonly dataStoreCount: number;
  readonly referencedDataStoreCount: number;
}
```

Reply:
btw I like that you found a way to move that `"ContainerLoadStats"` event out into `containerRuntime.ts` 👌

Reply:
Although this event will be logged by a different logger now... but they're identical, not sure why line 85 creates a new ChildLogger at all.  Oh well, not your problem.

Reply:
i agree with putting them into stats. my question is can i really do ...dataStores.stats in sendTelemetryEvent? I think I have to list individual properties one by one

Reply:
+1 to organize them into group. Unless there is good reason not to log some properties, I think it's Ok to log them all